tests/page-objects/default/sms/messages.wdio.page.js

```diff
@@ -45,6 +45,12 @@ const getMessageHeader = async () => {
   };
 };
 
+const getMessageHeaderOnMobile = async () => {
+  return {
+    name: await commonPage.mobileTopBarTitle().getText(),
+  };
+};
```

Comment:
We'll need this for other pages, and it's generic enough. Please move it to `tests/page-objects/default/common/common.wdio.page.js`
```suggestion
const getHeaderTitleOnMobile = async () => {
  return {
    name: await commonPage.mobileTopBarTitle().getText(),
  };
};
```

Reply:
Done